server/route/project.go

```diff
@@ -4,21 +4,129 @@ import (
 	"net/http"
 
 	"github.com/gin-gonic/gin"
+	"github.com/go-errors/errors"
+	"github.com/guregu/null"
+
+	"github.com/torinos-io/api/server/middleware"
+	project_service "github.com/torinos-io/api/service/project_service"
+	project_store "github.com/torinos-io/api/store/project_store"
 )
 
 // CreateProject creates project
 func CreateProject(c *gin.Context) {
-	c.JSON(http.StatusOK, gin.H{
-		"message": "Project created",
+	ac := middleware.GetAppContext(c)
+
+	projectStore := project_store.New(ac.MainDB)
+	service := project_service.New(project_service.Context{
+		Config:       ac.Config,
+		ProjectStore: projectStore,
 	})
+
+	request := &project_service.UploadRequest{}
+
+	{
+		header, err := c.FormFile("cartfile_content")
+		if err != nil {
+			c.Error(errors.New("Cartfile is empty"))
+			return
+		}
+
+		file, err := header.Open()
+		if err != nil {
+			c.Error(errors.Wrap(err, 0))
+			return
+		}
+
+		request.CartfileContent = file
+	}
+
+	{
+		header, err := c.FormFile("podfile_content")
+		if err != nil {
+			c.Error(errors.New("Podfile is empty"))
+			return
+		}
+
+		file, err := header.Open()
+		if err != nil {
+			c.Error(errors.Wrap(err, 0))
+			return
+		}
+
+		request.PodfileLockContent = file
+	}
+
+	name, _ := c.GetPostForm("repository_name")
+
+	request.RepositoryName = name
+
+	userID := null.Int{}
```

Comment:
Nitpicking but this can be done by:
```go
if user := middleware.GetCurrentUser(c); user != nil {
    request.UserID.Int64 = int64(user.ID)
    request.UserID.Valid = true
}
```